hw/js-lesson-6/scripts.js

```diff
@@ -0,0 +1,175 @@
+// На forEach
+
+// 1 Дан массив с числами. Создайте новый массив, состоящий из квадратов этих чисел. 
+
+let arr = [1, 2, 3, 4, 5];
+
+newArr = [];
+
+arr.forEach(function(element) {
+	newArr.push(element * element);
+    }
+);  
+
+console.log(newArr);
+
+
+// 2 Дан массив с числами. Найдите сумму этих чисел. 
+
+let arr = [1, 2, 3, 4, 5];
+let sum = 0;
+arr.forEach(function(element) {
+	sum += element;
+    }
+);
+
+alert(sum);
+
+
+// На map
+
+// 3 Дан массив с числами. Сделайте из него массив, состоящий из квадратов этих чисел. 
+
+let arr = [1, 2, 3, 4, 5];
+
+newArr = [];
+
+arr.map(function(element) {
+	newArr.push(element * element);
+    }
+);  
+
+console.log(newArr);
+
+
+// На every, some
+
+// 4 Дан массив с числами. Проверьте то, что все элементы в массиве больше нуля. 
+
+let arr = [1, 2, 3, 4, 5];
+
+let checkArr = arr.every(function(element) {
+	if (element > 0) {
```

Comment:
you don't need write it like that you could let a = arr.every(function(element) { return element > 0})